docs/source/alfacase_definitions/AnnulusDescription.txt

```diff
@@ -19,7 +19,7 @@
             has_annulus_flow: boolean
             has_annulus_flow: number
             pvt_model: string # optional
-            initial_conditions: \ :class:`initial_conditions_description_schema <InitialConditionsDescription>`\ # optional
-            gas_lift_valve_equipment: 
-                string: \ :class:`gas_lift_valve_equipment_description_schema <GasLiftValveEquipmentDescription>`\ # optional
+            initial_conditions: \ :class:`initial_conditions_description_schema <InitialConditionsDescription>` # optional
```

Comment:
Please review these escape characters, since it's making some attributes to be missing on docs.
![image](https://user-images.githubusercontent.com/5083518/114933752-6a4c1080-9e0f-11eb-897a-c7ca2a00856d.png)